package.json

```diff
@@ -63,5 +63,8 @@
   },
   "browserslist": [
     "extends browserslist-config-kolibri"
-  ]
+  ],
+  "volta": {
+    "node": "12.22.12"
+  }
```

Comment:
Although this volta version is necessary for your local dev environment to work properly, you don't want to commit it when you make changes.